vaadinfrontend/frontend/themes/datamanager/components/spreadsheet.css

```diff
@@ -0,0 +1,21 @@
+vaadin-combo-box-overlay::part(content) {
+  height: 30rem;
+  width: 20rem;
+  max-height: 30vh;
+}
+
+vaadin-combo-box-overlay::part(overlay) {
+  height: 30rem;
+  width: 20rem;
+  max-height: 30vh;
+}
+
+.spreadsheet-list-item {
+  display: grid;
+  grid-template-columns: auto 5%;
+}
+
+.spreadsheet-list-item > div{
+  width: 100%;
+}
```

Comment:
NIT: I'd move this to the div.css since you want to style css within a div component